resources/js/components/modules/meet/Meeting.vue

```diff
@@ -1,119 +1,96 @@
 <template>
     <div>
-        <b-tabs :small="true" v-model="tabIndex" >
+        <b-tabs :small="true" v-model="tabIndex">
             <b-tab title="👨‍👩‍👧‍👦 Reunión" :title-link-class="getTabClass(0)">
                 <!-- <router-view :meet="meet-activity"></router-view> -->
                 <Meet :user="user"></Meet>
             </b-tab>
-            <b-tab title="🎧 Audio!" :title-link-class="getTabClass(1)">
+            <b-tab title="🎧 Audio! " :title-link-class="getTabClass(1)">
                 <div class="jitsi-meet" id="jitsi-meet" align="center">
                     <div id="meet" style="margin-top: 25px">
-                    <vue-jitsi-meet
-                        ref="jitsiRef"
-                        domain="meet.jit.si"
-                        :options="startJitsiMeet"
-                >   </vue-jitsi-meet>
+                        <vue-jitsi-meet
+                            ref="jitsiRef"
+                            domain="meet.jit.si"
+                            :options="startJitsiMeet"
+                        >
+                        </vue-jitsi-meet>
                     </div>
                 </div>
             </b-tab>
         </b-tabs>
     </div>
 </template>
 <script>
-import { mapGetters } from 'vuex'
-import { JitsiMeet } from '@mycure/vue-jitsi-meet';
-import Meet from '../meet/Meet.vue'
+import { mapGetters } from "vuex";
+import { JitsiMeet } from "@mycure/vue-jitsi-meet";
+import Meet from "../meet/Meet.vue";
 export default {
     data: () => ({
         meet: null,
         tabIndex: 0,
+        userName: "",
     }),
 
-    props:['user'],
+    props: ["user"],
```

Comment:
Codacy has a fix for the issue: [Unexpected blank line between object properties.](https://app.codacy.com/gh/JarvinsonValencia/easyConference/pullRequest?prid=9473680)
```suggestion

```